aquarium/src/cg/group4/aquarium/Aquarium.java

```diff
@@ -0,0 +1,94 @@
+package cg.group4.aquarium;
+
+import cg.group4.client.Client;
+import cg.group4.data_structures.subscribe.Subject;
+import cg.group4.view.AquariumScreen;
+import cg.group4.view.StartScreen;
+import cg.group4.view.screen_mechanics.AssetsLoadingHandler;
+import cg.group4.view.screen_mechanics.LoadingScreen;
+import com.badlogic.gdx.Game;
+import com.badlogic.gdx.Gdx;
+
+/**
+ * The purpose of the Launcher is to launch the collection application.
+ * This Launcher is a separate launcher from the Launcher in the core project.
+ * This is because it does not require several pieces such as game logic.
+ */
+public class Aquarium extends Game implements AssetsLoadingHandler {
+
+    protected static volatile Aquarium instance;
+
+    /**
+     * Connects the client with the server.
+     */
+    protected Connector cConnector;
+
+    /**
+     * Loads the configuration for the server.
+     */
+    protected Configuration cAquariumConfig;
+
+    /**
+     * Screen which displays the aquarium.
+     */
+    protected AquariumScreen cAquariumScreen;
+
+    /**
+     * Screen which displays a textfield where the user can fill in the group id to show the aligned aquarium.
+     */
+    protected StartScreen cStartScreen;
+
+    /**
+     *
+     */
+    protected Subject cGroupIdSubject;
+
+
+    private Aquarium() {
+    }
+
+    public static Aquarium getInstance() {
```

Comment:
Why is this a Singleton? The only place I see you actually use this the getInstance is in the StartScreen. You can just pass it as a reference if you really need to have it there. Unless you use it often from different places there is no need for a Singleton.